packages/compass-web/src/entrypoint.tsx

```diff
@@ -298,7 +246,9 @@ const CompassWeb = ({
     []
   );
 
-  const connectionStorage = __TEST_CONNECTION_STORAGE ?? NoopConnectionStorage;
+  const connectionStorage =
```

Comment:
Should be a ref now, otherwise you're creating a new instance every render: re-rendering everything that's currently using connection store through hooks and locators